src/plugins/intel_gpu/src/graph/impls/ocl/arg_max_min.cpp

```diff
@@ -112,18 +112,19 @@ struct arg_max_min_impl : typed_primitive_impl_ocl<arg_max_min> {
 
 namespace detail {
 attach_arg_max_min_impl::attach_arg_max_min_impl() {
-    implementation_map<arg_max_min>::add(impl_types::ocl, arg_max_min_impl::create,  {
-        std::make_tuple(data_types::f32, format::bfyx),
-        std::make_tuple(data_types::f16, format::bfyx),
-        std::make_tuple(data_types::i32, format::bfyx),
-        std::make_tuple(data_types::i8, format::bfyx),
-        std::make_tuple(data_types::f32, format::bfzyx),
-        std::make_tuple(data_types::f16, format::bfzyx),
-        std::make_tuple(data_types::i8, format::bfzyx),
-        std::make_tuple(data_types::f32, format::yxfb),
-        std::make_tuple(data_types::f16, format::yxfb),
-        std::make_tuple(data_types::i8, format::yxfb),
-    });
+    auto types = {data_types::f16, data_types::f32, data_types::i8, data_types::i32};
+
+    auto formats = {format::bfyx,
+                    format::yxfb,
+                    format::b_fs_yx_fsv16,
+                    format::b_fs_yx_fsv32,
+                    format::bs_fs_yx_bsv16_fsv16,
+                    format::bs_fs_yx_bsv32_fsv16,
+                    format::bs_fs_yx_bsv32_fsv32,
+
+                    format::bfzyx};
+
```

Comment:
Now i32 is  added to all formats, which have not been supported... 
Is the support of i32 checked for those formats?

Reply:
yep, I've added i32 unit test to check part of them. The only part is just to avoid a big number of tests, but now I've checked locally and all the tests are passed. Should I add all the combinations to unit tests?

Reply:
https://github.com/openvinotoolkit/openvino/pull/12456/commits/ca92a6b55941ae1edaa51a1ec435a729067462c1#diff-41681deecec2e1f64490f442fe14044f695bdecfb57b7857f49770fb2e33aa79R53
it's here

Reply:
If you checked locally, then I believe it is okay :)